src/errorHandler.ts

```diff
@@ -100,7 +114,7 @@ export function safeExecution(
  */
 export function setupErrorHandler(): Disposable {
     // Setup the statusBarItem
-    statusBarItem = window.createStatusBarItem();
+    statusBarItem = window.createStatusBarItem(StatusBarAlignment.Right);
```

Comment:
You need to provide the second argument (priority) with non-zero value to avoid intermittent "jumping" of "Prettier" position back and forth. It looks like ESLint label has priority 0, so we need to set priority for Prettier to 1 or -1.

-1 would permanently move the "Prettier" label to the top-right corner.